projects/plugins/jetpack/modules/videopress/class.videopress-ajax.php

```diff
@@ -1,6 +1,7 @@
 <?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
 
 use Automattic\Jetpack\Connection\Client;
+use Automattic\Jetpack\VideoPress\Options as VideoPress_Options;
```

Comment:
Given `VideoPress_Options` is defined in a separate file (`class.videopress-options.php`) this should be fine 👍

Reply:
Yes, I don't include that file and name the new class as the rest of the code expects... I think it's fine. But if you think it's safer to rename it I'd do it